google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQuery.java

```diff
@@ -1176,4 +1176,7 @@ TableResult query(QueryJobConfiguration configuration, JobId jobId, JobOption...
    * @throws BigQueryException upon failure
    */
   TableDataWriteChannel writer(WriteChannelConfiguration writeChannelConfiguration);
+
+  // TODO(pongad): document
+  TableDataWriteChannel writer(JobId jobId, WriteChannelConfiguration writeChannelConfiguration);
```

Comment:
👍 

I assume you'll document before merging?

Reply:
Will do this also in a follow up